checkov/common/util/http_utils.py

```diff
@@ -42,9 +42,13 @@ def get_version_headers(client, client_version):
     }
 
 
+def get_user_agent_header():
+    return {'User-Agent': 'Mozilla/5.0 (KHTML, like Gecko) Chrome/85.0.4183.83 Safari/537.36'}
```

Comment:
just thinking, if we should change it to something, which doesn't look like a fake agent 😄 something like 
```suggestion
    return {'User-Agent': f'checkov/{checkov_version}'}
```

Reply:
If it works - let's do it